android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -99,12 +123,15 @@ public void getPermissionsStatus(final Promise promise) {
             return;
         }
 
-        boolean foreground = ActivityCompat.checkSelfPermission(activity, Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED;
+        boolean foreground = ActivityCompat.checkSelfPermission(activity,
+                Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED;
         boolean background = foreground;
-        boolean denied = ActivityCompat.shouldShowRequestPermissionRationale(activity, Manifest.permission.ACCESS_FINE_LOCATION);
-        
-        if (Build.VERSION.SDK_INT >= 29) {
-            background = ActivityCompat.checkSelfPermission(activity, Manifest.permission.ACCESS_BACKGROUND_LOCATION) == PackageManager.PERMISSION_GRANTED;
+        boolean denied = ActivityCompat.shouldShowRequestPermissionRationale(activity,
+                Manifest.permission.ACCESS_FINE_LOCATION);
```

Comment:
since we are now targeting 31, do we need to handle approximate location in android 12?

Reply:
@brettguenther I made some updates that align better with this comment, but it would be good to understand if there are larger needs around these checks (using https://github.com/radarlabs/radar-sdk-android/pull/106 as a reference point)

Reply:
ah nice - see Tim commented on what i was thinking as well with including ACCESS_COARSE_LOCATION